README.md

```diff
@@ -1,88 +1,9 @@
-# Ready release go :rocket:
+# plugin-ready-release-go
 
-This plugin can be executed on every push to your release branch (e.g. default branch) and will create a new release pull-request with an updated changelog as preparation for the next release. After merging the "release"-pull-request, a new release / tag will be created for you.
+[![Build status](https://ci.woodpecker-ci.org/api/badges/woodpecker-ci/plugin-ready-release-go/status.svg)](https://ci.woodpecker-ci.org/woodpecker-ci/plugin-ready-release-go)
+[![Docker Image Version (latest by date)](https://img.shields.io/docker/v/woodpeckerci/plugin-ready-release-go?label=DockerHub%20latest%20version&sort=semver)](https://hub.docker.com/r/woodpeckerci/plugin-ready-release-go/tags)
 
-## Usage
```

Comment:
We could link the docs.md or the plugin page for usage details.

Reply:
Idk, the docs are linked in GH already, if you prefer it to be linked in the readme as well I can do so

Reply:
It was not... Not it is

![grafik](https://github.com/user-attachments/assets/91e02cb1-aa3a-47bb-962f-40fa2f8c82e9)

Side-note: The URL generation for plugin pages is, lets call it weird 

![grafik](https://github.com/user-attachments/assets/2759f23c-a1bd-4385-b890-b57aa790728c)